specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2024-03-01/DataConnectors.json

```diff
@@ -651,6 +661,118 @@
       },
       "type": "object"
     },
+    "TiTaxiiDataConnector": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/DataConnector"
+        }
+      ],
+      "description": "Data connector to pull Threat intelligence data from TAXII 2.0/2.1 server",
```

Comment:
nit, doc style, probably need not capitalize threat, since I don't think its a proper noun in this context